adr/ADR-12.md

```diff
@@ -0,0 +1,133 @@
+# JWT library free jwt user generation
+
+|Metadata|Value|
+|--------|-----|
+|Date    |2021-07-20|
+|Author  | @mhanel, @aricart |
+|Status  |`Proposed`|
+|Tags    |client, jwt|
```

Comment:
better to have somthing like `accounts` or `security` instead of JWT - these tags are turned into indexes and its worth casting a slightly wider net so we dont end up with many single entry indexes

Reply:
changed to security